cerebrium/container-images/defining-container-images.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: Defining Container Images
+---
+
+## Introduction
+
+Cerebrium simplifies the deployment and running of machine learning apps by abstracting infrastructure management into configuration, allowing engineering teams to focus on what matters most - delivering value to customers using application code. A single TOML file manages environment setup, deployments, and scaling - tasks that typically require dedicated teams.
+
+## Why TOML?
+
+While Python decorators offer programmatic configuration, they scatter infrastructure settings throughout code files, making changes risky and review difficult. TOML centralizes all configuration in one place, making it easier to track changes and maintain consistency. Its straightforward syntax prevents the accidental complexity that often comes with code-based configuration, while its hierarchical structure naturally maps to modern app requirements.
```

Comment:
seems unnecessary

Reply:
Might seem unnecessary to us, but for users, one of the things they've mentioned is how easy it is to configure their apps on our platform and this is why. So this is more for users who might want to experience those same benefits

Reply:
I agree with eli that I think this is more a design design part of the platform that we should put in a different part - maybe in the how it works part?